lumigator/python/mzai/backend/backend/tests/api/routes/test_datasets.py

```diff
@@ -102,8 +110,8 @@ def test_presigned_download(app_client: TestClient, valid_experiment_dataset):
 
 def test_experiment_format_validation(
```

Comment:
What do you think of using mark.parametrize here instead of the for loop over datasets? 
You know the input you want to check (and it's just one at a time), you know the output you expect, so it could be:

```
@pytest.mark.parametrize(
    "dataset, expected_output", 
    [(missing, status.HTTP_403_FORBIDDEN),
    (extra, status.HTTP_403_FORBIDDEN)]
def test_experiment_format_validation(app_client, dataset, expected):
....
```

And it will become easy to add cases as we move into dataset checks later on.

Reply:
Oh this is nice! Added it, thanks for the suggestion 🙏